python/openinference-semantic-conventions/src/openinference/semconv/trace/__init__.py

```diff
@@ -148,6 +149,46 @@ class MessageAttributes:
     """
 
 
+class MessageContentAttributes:
+    """
+    Attributes for the contents of user messages sent to an LLM
+    """
+
+    CONTENT_TYPE = "content.type"
```

Comment:
```suggestion
    CONTENT_TYPE = "message_content.type"
```

Reply:
Better to. be explicit - otherwise it implies it can be top level

Reply:
I find this a bit weird. Drawing a comparison with tool calls, we would on one hand have:
```
message.tool_calls = [{"tool_call.function.name": "get_current_weather",...}]
```
But for contents we would have:
```
message.content = [{"message_content.type": "text",...}]
```
Seems repetitive to have the message there no?

Reply:
hmmm. I see. My problem with it is mainly that `content.type` needs to be valid top level.